.circleci/config.yml

```diff
@@ -272,14 +272,24 @@ workflows:
           ruby_opt: -W:deprecated
           <<: *important-branches
       - tests_macos:
-          name: 'Execute tests on macOS (Xcode 13.0.0, Ruby 3.0)'
-          xcode_version: '13.0.0'
+          name: 'Execute tests on macOS (Xcode 13.4.1, Ruby 3.0)'
+          xcode_version: '13.4.1'
           ruby_version: '3.0'
           ruby_opt: -W:deprecated
           <<: *important-branches
       - tests_macos:
-          name: 'Execute tests on macOS (Xcode 13.0.0, Ruby 3.1)'
-          xcode_version: '13.0.0'
+          name: 'Execute tests on macOS (Xcode 13.4.1, Ruby 3.1)'
+          xcode_version: '13.4.1'
+          ruby_version: '3.1'
+          ruby_opt: -W:deprecated
+      - tests_macos:
+          name: 'Execute tests on macOS (Xcode 14.3.1, Ruby 3.1)'
+          xcode_version: '14.3.1'
+          ruby_version: '3.1'
+          ruby_opt: -W:deprecated
+      - tests_macos:
+          name: 'Execute tests on macOS (Xcode 15.0.0, Ruby 3.1)'
+          xcode_version: '15.0.0'
```

Comment:
There are some issues with SSL when using Ruby 3.1 😬 let's see if this PR can pass without adding new Xcode versions... 🤔 

```suggestion
```